src/test/java/com/ericsson/ei/subscription/SubscriptionHandlerTest.java

```diff
@@ -116,13 +116,13 @@
     private MockMvc mockMvc;
 
     @MockBean
-    private SendHttpRequest springRestTemplate;
+    private HttpRequestSender springRestTemplate;
```

Comment:
rename springRestTemplate to httpRequestSender

Reply:
Will fix this